src/ontology/uberon-edit.obo

```diff
@@ -80988,6 +80988,8 @@ intersection_of: part_of UBERON:0000079 ! male reproductive system
 disjoint_from: UBERON:0005056 ! external female genitalia
 disjoint_from: UBERON:0009196 ! indifferent external genitalia
 relationship: contributes_to_morphology_of UBERON:0000079 ! male reproductive system
+relationship: present_in_taxon NCBITaxon:50557 ! Insecta
+relationship: present_in_taxon NCBITaxon:9606 ! Homo sapiens
```

Comment:
> Add present_in_taxon statements for male and female external genitalia to ensure they work in both mammals and insects.

Wouldn't it be more accurate to use [Mammalia](http://purl.obolibrary.org/obo/NCBITaxon_40674) instead of Homo sapiens?

Reply:
For present_in_taxon it's useful to be as specific as possible, as far as QC goes (present in Mammalia is implied). Any number of other species could be added. But if the intent is just readability that would be different.